hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -216,3 +226,12 @@ def gen_hash(content_id: str) -> t.Optional[HashResult]:
         "content_hash": record.content_hash,
         "updated_at": record.updated_at.isoformat(),
     }
+
+
+def gen_hash_count() -> t.Dict[str, int]:
```

Comment:
nit: gen_ usually indicates that this is an asyncronous our yielding function, suggest renaming

Reply:
following a bad pattern of mine, removed the use of gen in #470